easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -233,6 +238,20 @@ def configure_step(self):
         else:
             make_ext = '.sys'
 
+        # patch Modules/parameter.f90 file to set compilation parameters
+        regex_para = []
+        if self.cfg['ntypx']:
+            regex_para.append((r"ntypx\s*=\s*\S+", r"ntypx = %s," % self.cfg['ntypx']))
+        if self.cfg['nsx']:
+            regex_para.append((r"nsx\s*=\s*\S+", r"nsx = %s," % self.cfg['nsx']))
+        if self.cfg['npk']:
+            regex_para.append((r"npk\s*=\s*\S+", r"npk = %s," % self.cfg['npk']))
+        if self.cfg['lmaxx']:
+            regex_para.append((r"lmaxx\s*=\s*\S+", r"lmaxx = %s," % self.cfg['lmaxx']))
+        if self.cfg['lqmax']:
+            regex_para.append((r"lqmax\s*=\s*\S+", r"lqmax = %s" % self.cfg['lqmax']))
```

Comment:
@Danzelot Looks like this can be cleaned up even more, it's basically a for loop. And you can simplify the regex too using `.*`:

```python
for key in ['lmaxx', 'lqmax', 'npk', 'nsx', 'ntypx']:
    if self.cfg[key]:
        regex_para.append((key + r'\s*=.*', key + ' = ' + self.cfg[key]))

Reply:
Nice suggestion.
I had to change the regex a little to not mess up the fortran90 parameter file.
Should all work now. ( At least on my system)